### PR TITLE
To do list  list structure

### PR DESCRIPTION
**Pull request by hafiz1379**
In this project I:

- [ ] Set up a new project with webpack that is based on the webpack exercise I have already completed.
- [ ] Create an index.html file and write my HTML markup here.
- [ ] Create an index.html file and write your HTML markup here.
- [ ] Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- [ ] On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- [ ] Create a style.css and set rules for the To Do List

All my source files (index.html, index.js and style.css)  located in /src directory and my distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+addButton.addEventListener('click', () => {
+  const description = addInput.value;
+  if (description) {
+    Task.addTask(description);
+    addInput.value = '';
+    renderTasks(Task);
```
Comment (Shisui6) [inline src/index.js:14]: According to the project requirements, by default, your app should contain a hardcoded array with some simple to-do tasks containing three keys. Then the tasks are used to populate a HTML list item. Please update your application to include this feature.

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @hafiz1379, 

# Requires Changes 🍀
You have done a great job so far!👏👏
I noticed that your to-do list app is fully functional. Please note that for each project, you are supposed to implement the functionality in each milestone step by step so that you learn something new every day. Although your app is complete, it does not fulfil the requirements set for this project

## Highlights🎯
- Webpack setup properly ✅
- App works as expected ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -1,21 +1,48 @@
...
+      }
+    });
+  }
 }
-
-document.body.appendChild(component());
\ No newline at end of file
+window.onload = () => {
+  const todoList = document.querySelector('.todo-list');
+  Task.tasks.forEach((task) => {
```
Comment (OyePriscilla) [inline src/index.js:36]: - [optional] On page load, the task array is rendered dynamically to the DOM and the tasks should appear in order of the index values for each task. **Please kindly implement this functionality.**

**Review by OyePriscilla — APPROVED** [review]
Hi @hafiz1379 👏 👏 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Rendered the task array i dynamically to the DOM ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**